packages/main/src/plugin/index.ts

```diff
@@ -120,15 +123,52 @@ export class PluginSystem {
     });
   }
 
+  // Create an Error to access stack trace
+  // Match a regex for the file name and return it
+  // return nothing empty string if file name not found
+  async getExtName(){
+
+    const stack = new Error().stack
+
+    var validExt:string[] = [];
+
+    if(this.extensionLoader != undefined){
+
+      //Take the id information from extensionLoader
+      let validExtObj = await this.extensionLoader.listExtensions();
+
+      validExtObj.forEach((extInfo: any) => {
+        validExt.push(extInfo.id);
+      })
+    }
+    
+    if(validExt.length === 0)
+      return 
+    if(stack){
+      var insert = validExt.join('|')
+      const re = new RegExp(`(podman\-desktop\\\\extensions)\\\\(${insert})`)
+      var extensionGroupings = stack.match(re)
+      if (extensionGroupings != null)
+      return '[' + extensionGroupings[2] + ']';
```

Comment:
we need to display the extension name, not the id, so we should keep the mapping extension path<--> extension name